SignInCheckIn/SignInCheckIn/Services/EventService.cs

```diff
@@ -80,6 +80,33 @@ public EventDto GetCurrentEventForSite(int siteId)
         //    return Mapper.Map<MpEventDto, EventDto>(currentEvents.First());
         //}
 
```

Comment:
It was obsolete because the stored proc was supposed to be updating the cancelled flag but it was not. I moved it to the EventService because I thought that made more sense